src/AzureStack/Subscriptions/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@bganapa please do not make Resource Provider name plural.
This should be Subscription.
This should be changed across the namespace, directory name, package name, project name.

Reply:
The guidance is to match the RP namespace I believe. In this case it is Microsoft.Subscriptions rather than Microsoft.Subscription... hence the plural

Reply:
@shahabhijeet We have already merged another RP that is plural and in fact there are multiple RPs in Azure that have plural namespaces/directory names/package names/project names.  Why is this PR being treated any different than those RPs?  Also, why did the REST team not flag this as not allowed if that is the case?  Please point us to the requirement and we will start an email thread as needed to ask for an exception.

Reply:
The RP Namespace is Microsoft.Subscriptions, this should reflect that.